tests/tracking/fluent/test_fluent.py

```diff
@@ -917,3 +917,19 @@ def test_delete_tag():
     with pytest.raises(MlflowException, match="No tag with name"):
         mlflow.delete_tag("b")
     mlflow.end_run()
+
+
```

Comment:
could you add a validation test here for behavior of if the description tag is set in both the manually-provided dict of tags and a description is set to ensure that an exception is thrown for key collision?